src/main/java/org/opentripplanner/graph_builder/module/ned/NEDDownloader.java

```diff
@@ -1,559 +0,0 @@
-package org.opentripplanner.graph_builder.module.ned;
```

Comment:
Is this not used any more? I guess downloading elevation data each time a graph is build is not an appropriate thing to do. So, I am in favor of removing it.